src/Docker.DotNet/Endpoints/SwarmOperations.cs

```diff
@@ -157,43 +156,8 @@ async Task<ServiceUpdateResponse> ISwarmOperations.UpdateServiceAsync(string id,
             return this._client.JsonSerializer.DeserializeObject<ServiceUpdateResponse>(response.Body);
         }
 
-        public Task<Stream> GetServiceLogsAsync(string id, ServiceLogsParameters parameters, CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
Is this method unused?

Reply:
I must have erroneously deleted those during merge